mkt/site/tests/test_monitor.py

```diff
@@ -1,20 +1,68 @@
 import time
 
 from django.conf import settings
+from django.test.utils import override_settings
 
 from mock import Mock, patch
-from nose.tools import eq_
+from nose.tools import eq_, ok_
+import redisutils
 import requests
 
 import mkt.site.tests
-from mkt.site.monitors import receipt_signer as signer, package_signer
+from mkt.site import monitors
 
 
 @patch.object(settings, 'SIGNED_APPS_SERVER_ACTIVE', True)
 @patch.object(settings, 'SIGNING_SERVER', 'http://foo/')
 @patch.object(settings, 'SIGNED_APPS_SERVER', 'http://baz/')
 class TestMonitor(mkt.site.tests.TestCase):
-    # Some rudimentary tests for the rest of the monitor would be nice.
+    # Would be nice to tests what happens when test_memcache(),
```

Comment:
typo s/test/test/

Also should this have a TODO: prefix so it can be more easily collated with all the other TODOs - or better still a bug?